docs/source/onboarding_resources/tutorial/index.ipynb

```diff
@@ -0,0 +1,1653 @@
NnbV2rBVfMPo9bq/f0/sXssZp3rfy07MS31nvJPUWqvgclhg0VMXnRY1Wuzcm1SrsW5uW6LHYF748bLilrEonzN7VsH7P4e2L8WqCmTN5ogKRMYdCGQqxxx9Y9iq7IfxV7T1zhgXg4rvEdS+iM2csiJOhPdjWciSpMsu9TeMIQjokjEEI6IyAdDuMAxhKNIdbTKhovfK/AKoMQgtUVXZaJrUsNrZjWmstSEF+5fjoLdddddy+yVjDv/NQHxyaEx/XLp3mrc93XdjRruG5OC64YkyqplqhxmmHxCOnH9ee0OvFEjRlR5ezbxIpwd1VtWFBjl6/rIdvcouaauI6fROndZFaPjeo5Vav+BsdhFNd+yFvtN36LY9qvs1k8DLboYTlXXjuukH6h2qP1gCEdEkYghHBGRD4ZwgWMIR5Hqz0uP4fNdVbJyumZwIu4fmyKrwJhrbXj14ZXY9cNR2XFLSY9Vg7i07KZt/BBs/15dgrd/KZeVkwgk35zaGYPTWz8stMOBW0o/xFrLQdlxioIeC1OvQ3dd6/wdkIeaEmDPKmcgt28tYK2VNxoQmwoMPB8YfIlzc4d6VNoPY69pMQ6aVzZpdFysthPS9UOQou+JOG2aUndEjIabc4QyhnBEFIkYwhER+WAIFziGcBSJdhwz48r5h2XllBClxVfXZSHO0LpvKt/5x3psWHxAVm4x8Qb85h/j0CP35C9gb1e+/W/9+Ah+POy9Q2xKjBYfTM9AWlzrT5+tcJhwafE7yLd7T5nM0ibho1Ru1BBUpkrlm2AxsOVz4Mg22WxE1inAoIuBvhOda8n54R4d9x2Kbbtkt2nESLkYbQc1kBMhnfMQ17JWQzoGPicLQzgiikQM4YiIfDCECxxDOIpE1390BJsKvQOnP09IxfSBwRmZ9vmcLVj89nZZeeuYGY/B4zKROyYDPQefvECuzGRXN2oorPRezy03zYh3Lusiq9a1z1aCqcVvwwTvzSG4UUMbEhs7iDBu21dAlfcuwX4ZY4F+k4GBFwCp3YBo/1OWK+wF2GdaioPmFU0aHdcUYrRcnC5N3Xk1Vjkn6boiUZuFOG26/AgKFoZwRBSJGMIREflgCBc4hnAUCSrMDizdW4Ule6qx8mDdQKBnigEfXpkhq+BY/elefDCr7jponsTouEHjMtVQrt/wdBii2nYDh21FZly9wHuEoDBtYAIenpAqq9a1xLwbd5V9Iis3btTQxhw2YP96YOvnwO7lgM0ibzRAq3x9ivXjBpyrfBONq3eE3AHz9+rR3NFxTaVDFJJ02cqRg2Tl6KDvi3htcILjSMUQjogiEUM4IiIfDOECxxCOwpUY2SU2HVi8uwpr8xte/+qVi9NxWma0rIJn27ojeP3RVbCa7bJTP4NRh77D0zFobIZ6xCW1zfTMj7ZX4ollx2Xl9teJHXFR3zhZta7ZVSvxSvV6Wbn9N+liTDL2khW1GVMFsP0bZyDX1OmqRuVro89EYOB5QNZQpVE3nHGOjlvS5LXjAmHUJKKDvg866JRD3xfJuu7qlFdqGYZwRBSJGMIREflgCBc4hnAUToqqbVi2TwRv1dhwqAkLzytOz4nBs+elySr4Du4swYv3L0d1hVl2Gie+TXMGdsCgsZkYenoWOnQJThjm8viy41i4vVJWTkadBm9f1gV9Oxhkp/WIZ5+bSxf43ahhfurV6K07+evmRaziA8CWz4BtXwJVdcNZvxLSgf5nA7kXAildZdOTAzWOEtTYj6HKXiTPx9RztVJX24+r68u1Jh2M6kYQHXR91VAuVd9b+eoKfvAeLhjCEVEkYghHROSDIVzgGMJRe5dXblVHvIljs886b40Z2zUGD41PRVaiXnbahrnWiu3rj+CX5QXYtu4waiqbFzh0zknEwFFdMHxyN2T0SJLd1nXVgsPYXuQdFHaO1+GDyzOQFNX6I4rq26ihszYBn6Rez40aTjbXdFWxftyeJk5XFdKUn8f9z3VOWY1t+q63tY5Sj5Du+ImwrloGdTY0PcT2R4yKE2FchmEEuhiGI0YTnOnW4YIhHBFFIoZwREQ+GMIFjiEctUdid9NvRfC2rxp7ipseYBm0wOjsGEzuGYczu8ci3njy30A67A7s3XIcW1YXYPOqQyjK9x6B1pgRk7vh4tsHI7FD647qOVplw7QPDqHc5D11dliXKLx2SWdog/DQiY0appW8g2qH998pN2oIMeZqYPvXztFxhzbLZiM0yjdf1+HOMK73Gco3Y4y80VIOlNkO4rhtJ45bnYcI7lpKrCXXxTACGYbhSNBmyi65MIQjokjEEI6IyAdDuMAxhKP2Zk1eLe74rFBWjRPTKMWIt0k9YtXgLdYQ2m8aj+ZVqGHcltWHsG/LMeV7U95ogDFah0lX9cPEK/uqa8m1lh8OmXDrx0fU6aKebjolEfeMavqopuZYbt6H28sWysrt1tgRuI8bNYSeskPu6arlR2SzEWIDh4HnO3dZFQzRQGwqkNjZWbdQpf0wSm37UWLbgxLrbuX6QIumtYrdVp0j5E5Fqq637EY2hnBEFIkYwhER+WAIFziGcNTePLz0GD7bVSUr/6L1zuBtcs9YnJ4Tixilbo+qy83YsuaQGsrt2FgIc41V3vEvJS1WHRU3bGK27ATufz+XY9aaElm5/ef8NEzoFuhoJv/+W7UaL1SvlZUbN2oIZcrPkPyfndNVd30HWKplv5miE4G4VCC2g3IWh3ItziKkU2t5xCYrH9zw97XyCgHl9nyUWPegxLYXpbY9KLflq/2mitIkq2GcGCHXST9A+S+27Y7FoYIhHBFFIoZwREQ+2kMIN3XqVFxzzTWYNm2a7NS1bt06jBo1SlZOubm52LzZOc1HvMidNWsWZs6cqdaeZs+ejXvvvbfFIRpDOGpPTDYHzpiThxpr3a9XEbyJTRYmdo9Tz6ION9s3HMHW1YewaeUhlB2rf3fJ7rkdcNldpyC7T+uMVntgcREW7/EOVcTj+/70LuieHJyNGm4v+wgrzPudD
```

Comment:
In the bullets at the top about the UC Davis data you state "31% lower prevalence of severe wasting" which would kinda undercut this line. I don't think it's wrong to make this assumption still, but we do seem to have some data supporting the concept of greater effects in more severe cases.